### ISSUE TITLE
05-Sync-Unicorn fails with permission issue

### ISSUE THREAD

**Issue by martin-bootstrap** [Sitecore/Habitat#283]
This is the error I get when I try to run gulp. I've double checked everything and followed the directions generally as per these steps:

1) install new sitecore instance using SIM 8.2. update 1
2) install WFFM using SIM 
3) ran post-install steps for WFFM and verified results
4) Updated configs on clean Habitat download
5) npm install && gulp - everything works fine until the unicorn syncing

```
Sync-Unicorn: Preparing authorization for http://habitat.dev.local/unicorn.aspx?verb=Sync&configuration=


Error: Invoke-WebRequest :

Error:
        Access to the path
'C:\path\to\projects\Habitat\src\project\habitat\serialization\Project.Habitat.Users' is denied.
```

I dug into the serialization folder above and noticed that Project.Habitat.Users is altogether missing. I'm sure this has something to do with the error. Please advise. Thanks.

**Comment by martin-bootstrap** [comment]
A work around for now was to edit the ``Habitat.Website.Serialization.config`` file and comment out the ``userDataStore``, ``userPredicate``, and ``userSyncConfiguration`` elements. I suspect this will need to be reenabled to handle.

**Comment by Eldblom** [comment]
Can you run this from /unicorn.aspx? Are you running Visual Studio or the commend line as an administrator? If you are not using the default (`c:\projects`, `c:\websites`, `habitat.dev.local`) Have you verified that all paths and domains are set up correctly in all config files as per the documentation?

**Comment by MaKsaw** [comment]
I ran into the same issue when setting up habitat solution for the first time. I was running Visual Studio as administrator. In my case, the folder I checked-out habitat solution into had _restrictive_ file/folder permissions. Then I setup the solution in a different folder (different to the defaults) and it worked fine.
I noticed the folder in question (C:\path\to\projects\Habitat\src\project\habitat\serialization\Project.Habitat.Users) does not exist in github. I think some process creates that folder post source control check-out. 
I guess that process is IIS worker process. So worth trying after granting IIS app-pool user permission to create folders in project folder (i.e c:\projects)
